### DISCUSSION TITLE
Add multi-language support

### DISCUSSION THREAD

**Discussion by vpsad** [plausible/analytics#2104] [Anything Else]
Plausible Analytics is the best lightweight statistical project I've seen so far, and it can be self-hosted.
So I think such a good project should also support multiple languages for better internationalization.
For example: Chinese, Japanese, Indian, Spanish, Portuguese, etc.

**Comment by RobertJoonas** [comment]
Thanks for the feedback and kind words, @vpsad! Agreed, would be great to use Plausible in languages other than English as well. We'd love to add that in the future. Unfortunately, there are no short term plans to work on this as we're a small team with limited resources. Thank you for understanding!

By the way, we have another feature thread for this with some more discussion. See: https://github.com/plausible/analytics/discussions/354
